src/main/java/pl/agh/edu/master_diet/service/AchievementConverter.java

```diff
@@ -0,0 +1,23 @@
+package pl.agh.edu.master_diet.service;
+
+import org.springframework.stereotype.Component;
+import pl.agh.edu.master_diet.core.model.database.Achievement;
+import pl.agh.edu.master_diet.core.model.rest.AchievementsResponse;
+import pl.agh.edu.master_diet.service.converter.GenericConverter;
+
+@Component
+public class AchievementConverter implements GenericConverter<Achievement, AchievementsResponse> {
+    private final static int NO_PROGRESS = 0;
```

Comment:
You sometimes add a newline after class name and sometimes not, please follow the convention

Reply:
Done.